src/hooks/useBankTransactions/useAugmentedBankTransactions.tsx

```diff
@@ -146,19 +144,20 @@ export const useAugmentedBankTransactions = (
     size,
     setSize,
   } = useBankTransactions({
-    startDate: filters?.dateRange?.startDate,
-    endDate: filters?.dateRange?.endDate,
-    tagFilterQueryString: filters?.tagFilter ? tagFilterToQueryString(filters.tagFilter) : undefined,
-    direction: filters?.direction?.length === 1
-      ? filters.direction[0] === Direction.CREDIT
-        ? 'INFLOW'
-        : 'OUTFLOW'
-      : undefined,
     categorized: filters?.categorizationStatus
       ? filters.categorizationStatus !== DisplayState.all
         ? filters.categorizationStatus === DisplayState.categorized
         : undefined
       : undefined,
+    descriptionFilter: filters?.descriptionFilter,
+    direction: filters?.direction?.length === 1
```

Comment:
I know you didn't write this but we should just change the client-side enums to use `INFLOW` and `OUTFLOW`. That can be later though.